src/components/AiMarketplace/MainSection/ServiceCollection/ToolBar/index.js

```diff
@@ -24,8 +24,7 @@ const ToolBar = ({ listView, total_count, handleSearchChange, toggleView }) => {
   return (
     <Grid container spacing={24} className={classes.toolBar}>
       <Grid item xs={12} sm={6} md={6} lg={6} className={classes.sortBySection}>
-        <span className={classes.sortbyTxt}>Sort by:</span>
-        <StyledDropdown labelTxt="" />
+        <Sort />
```

Comment:
This could be named as `ServiceSortOptions`. `Sort` sounds more like an action.

Reply:
https://github.com/singnet/refactored-dapp/pull/16/commits/d50b6aebd40b4d4ea9857a7c89724ace92a806bd